src/Engine/ProtoCore/Parser/atg/Associative.atg

```diff
@@ -1055,14 +1055,30 @@ Associative_DecoratedIdentifier<out ProtoCore.AST.AssociativeAST.AssociativeNode
                         .)
         ':'
 
-        ident
                         (. 
-                            int type = core.TypeSystem.GetType(t.val); 
+                            string ident = string.Empty;
+							int type = ProtoCore.DSASM.Constants.kInvalidIndex;
+							if (la.kind == 1)
+							{
+								if (IsIdentList())
+								{
+									Associative_IdentifierList(out node);
```

Comment:
So i guss the codegen will get an identlist as the LHS, hence no codegen changes?